src/qml/filters/speed/meta_forward.qml

```diff
@@ -0,0 +1,33 @@
+import QtQuick 2.0
+import org.shotcut.qml 1.0
+
+Metadata {
+    type: Metadata.Link
+    objectName: 'speedForward'
+    isAudio: false
+    name: qsTr("Speed (Forward)")
+    keywords: qsTr('temporal speed ramp fast slow motion', 'search keywords for the Speed filter') + ' speed'
+    mlt_service: "timeremap"
+    qml: "ui_forward.qml"
+    isFavorite: false
+    allowMultiple: false
+
+    keyframes {
+        allowTrim: false
```

Comment:
You need to bump the version in `link_timeremap.yml` and require it here using `minimumVersion`.

Reply:
I already bumped the version in `link_timeremap.yml`.  JB pleaded for a MLT release for kdenlive. So, I did that. New version number is 2.

Reply:
Done